runtime/gc_glue_java/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -465,6 +486,11 @@ MM_CollectorLanguageInterfaceImpl::scavenger_backOutIndirectObjectSlots(MM_Envir
 		while((slotPtr = classStaticsIterator.nextSlot()) != NULL) {
 			_extensions->scavenger->backOutFixSlotWithoutCompression(slotPtr);
 		}
+		GC_ConstantPoolObjectSlotIterator constantPoolObjectSlotIterator((J9JavaVM*)_omrVM->_language_vm, classToScan, true);
+		while ((slotPtr = constantPoolObjectSlotIterator.nextSlot()) != NULL) {
+			_extensions->scavenger->backOutFixSlotWithoutCompression(slotPtr);
+			printf("GC!!\n");
```

Comment:
printf

Reply:
sorry I just made some changes earlier this day and I'm still testing it